profiler/README.md

```diff
@@ -255,7 +255,7 @@ Note: Column to image kernel adds to the output memory, this will cause output b
 ./bin/ckProfiler permute_scale        0       1     1    0     1    64   64   64       4096         64          1           1          64        4096
 ```
 
-Result (MI100, FP32)
+Result:
```

Comment:
Here and above: Maybe remove these "Best perf" lines? I think there is no point in keeping such values here without any context

Reply:
Yeah, I think we can probably remove the sample output altogether.